src/Data/Bin/Properties.agda

```diff
@@ -26,16 +26,16 @@ import Relation.Nullary.Decidable as Dec
 open import Relation.Nullary.Negation using (contradiction)
 open import Algebra.FunctionProperties {A = Bin} _≡_
 open import Algebra.Structures {A = Bin} _≡_
-import Algebra.Properties.CommutativeSemigroup ℕₚ.+-semigroup ℕₚ.+-comm
-  as ℕ-+-semigroup
-import Algebra.Properties.CommutativeSemigroup ℕₚ.*-semigroup ℕₚ.*-comm
-  as ℕ-*-semigroup
+import Algebra.Properties.CommutativeSemigroup ℕₚ.+-semigroup ℕₚ.+-comm as
+  Of+ℕ-semigroup
```

Comment:
As requested, please get rid of these changes.

Reply:
Returned
```
import Algebra.Properties.CommutativeSemigroup ℕₚ.*-semigroup ℕₚ.*-comm
  as ℕ-*-semigroup
```

Reply:
This change doesn't seem to have happened. Could you double check?

Reply:
There was some confusion. I hope, now it agrees with the requirement.
Does it?

Reply:
No, unfortunately not.